### PR TITLE
New typings for module wtfnode

### PR DESCRIPTION
**Pull request by dex4er**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by RyanCavanaugh** [comment]
@dex4er Please fix the failures indicated in the Travis CI log.

File: types/wtfnode/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": false,
```
Comment (andy-ms) [inline types/wtfnode/tsconfig.json]: This should be `true` for all new packages.

File: types/wtfnode/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+// Type definitions for wtfnode 0.5
+// Project: https://github.com/myndzi/wtfnode
+// Definitions by: Piotr Roszatycki <https://github.com/wtfnode>
```
Comment (andy-ms) [inline types/wtfnode/index.d.ts]: This should be `https://github.com/dex4er` not `wtfnode`

File: types/wtfnode/tsconfig.json (hunk 1 above)
Comment (dex4er) [reply types/wtfnode/tsconfig.json]: I hate dts-gen :)

**Pull request merged**